src/renderers/testing/ReactTestRendererFiber.js

```diff
@@ -16,8 +16,19 @@
 var ReactFiberReconciler = require('ReactFiberReconciler');
 var ReactGenericBatching = require('ReactGenericBatching');
 var emptyObject = require('emptyObject');
+var ReactTypeOfWork = require('ReactTypeOfWork');
```

Comment:
I've confirmed that `build/packages/react-test-renderer/lib/ReactTypeOfWork.js` exists